examples/getting_started/__init__.py

```diff
@@ -0,0 +1,3 @@
+#
```

Comment:
Do we need this if we're moving `openbot_env.py` into a `common` folder?

Reply:
We will need it in the `common` folder, but not here.